### ISSUE TITLE
Looks like incorrect order of arguments for binary_crossentropy loss

### ISSUE THREAD

**Issue by a4tunado** [keras-team/keras#8013]
https://github.com/fchollet/keras/blob/master/keras/losses.py#L57 
https://github.com/tensorflow/tensorflow/blob/r1.3/tensorflow/contrib/keras/python/keras/backend.py#L2940

**Comment by farizrahman4u** [comment]
Order is different for standalone Keras and tf.contrib.keras (@fchollet Why?), but each implementation is consistent with itself. See [tf.contrib.keras.losses.binary_crossentropy](https://github.com/tensorflow/tensorflow/blob/r1.3/tensorflow/contrib/keras/python/keras/losses.py#L78), the arguments are reversed.

**Comment by harshini-gadige** [comment]
Hi

This issue isn't related to a bug/enhancement/feature-request or other accepted types of issue.

To ask questions, please see the following resources :

https://keras.io/
https://keras.io/#support
https://gitter.im/Keras-io/Lobby

Thanks!

If you think I made a mistake, please open another issue.
